core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -420,32 +420,31 @@ public ServerBuilder service(PathMapping pathMapping, Service<HttpRequest, HttpR
     }
 
     /**
-     * Binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}
+     * Binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}s
      * of the default {@link VirtualHost}.
      *
      * @throws IllegalStateException if the default {@link VirtualHost} has been set via
      *                               {@link #defaultVirtualHost(VirtualHost)} already
      */
     public <T extends ServiceWithPathMappings<HttpRequest, HttpResponse>>
-            ServerBuilder service(T serviceWithPathMapping) {
-        return service(serviceWithPathMapping, Function.identity());
+            ServerBuilder service(T serviceWithPathMappings) {
```

Comment:
This indention is different with  service(T serviceWithPathMappings) in HostBuilder, is it OK?